product_docs/docs/pgd/5/routing/proxy.mdx

```diff
@@ -48,6 +48,64 @@ Note: In this case, if the grace period is kept too high then writes will contin
 
 Having said that, most of the time, upon loss of the current Raft leader the new Raft leader gets elected by BDR within a few seconds provided more than half of the nodes (quorum) are still up. Hence, if the Raft leader is down but the write leader is still up then proxy can be configured to allow routing by keeping `consensus_grace_period` to a non-zero, positive value. The proxy will wait for the Raft leader to get elected during this period before stopping routing. This may be helpful in some cases where availability is more important.
 
+### Proxy health check
+
+PGD Proxy provides the following HTTP(s) health check API endpoints. These APIs are `GET` requests and do not require a request body.
+```
+GET /health/is-live
+GET /health/is-ready
+```
+
+#### Liveness
```

Comment:
let's move it after readiness... this is not important endpoint and may unnecessarily raise question

Reply:
done